tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -14,7 +14,7 @@
     "We introduce here a new mode of interaction between the atoms driven by the dipole-dipole interaction. This mode is called the XY mode. The interaction hamiltonian of a system of $N$ qubits is now given by \n",
     "\n",
     "$$\n",
-    "H_{XY} = \\frac{1}{2}\\sum_{i<j} \\frac{C_3}{R_{ij}^3}(\\sigma^+_i \\sigma^-_j + \\sigma^-_i \\sigma^+_j) \n",
+    "H_{XY} = \\sum_{i=1}^N\\sum_{j<i} \\frac{C_3}{R_{ij}^3}(\\sigma^+_i \\sigma^-_j + \\sigma^-_i \\sigma^+_j) \n",
     "$$\n",
     "\n",
     "where $\\sigma^{±} = (\\sigma^x ± i\\sigma^y)/2$, $R_{ij}$ is the distance between atom $i$ and atom $j$, and $C_3$ is a coefficient dependent on the device and taken as  $\\frac{C_3}{\\hbar} = 3700 \\frac{\\mu m^3}{\\mu s}$ for the `MockDevice`.\n",
```

Comment:
Won't it be clearer to have the units not as a fraction ?

Reply:
To me it doesn't make a big difference, I'll change it